### DISCUSSION TITLE
How to get the full program (by Google Translate)

### DISCUSSION THREAD

**Discussion by ghost** [Ross-Patterson/Portals-Desktop-Organization#4] [General]
Hi, sorry I can't speak English natively but I would like to enjoy the full Portals Desktop Organization program as I consider it the best ever, even in singular form.
There would be a way to obtain the license in another way, as the value in another currency is confusing for contribution.
If it is not possible, disregard this email and I will be happy for the development of this great program for windows.
I'm from Brazil and I'm going to recommend the program in the different types that I promote. Grateful!

Note: I replicated this same question in the email that appears on your website.

**Comment by Ross-Patterson** [comment]
Hi @LucRoGH, currently the only way to get a license key is here: https://rosspat.gumroad.com/l/xUGICq

I have emailed you a free license key to use, to save you any further problems.

**Reply by ghost** [reply]
Hi Ross...
I don't even know how to thank you!
I can only wish you success in your life.
I'm working on youtube projects or another platform and I won't forget you and your magnificent program!
God bless you!
Thanks.
